### ISSUE TITLE
improve `new_database` subcommand performance when targeting postgresql db

### ISSUE THREAD

**Issue by dhoogest** [fhcrc/taxtastic#139]
The `new_database` subcommand works great for sqlite dbs, however when a postgres db is the target, insertion of the large names/nodes tables is extended by an order of magnitude, most likely due to sqlalchemy+psycopg2 `executemany` nuances. Here's the timing on my fairly wimpy desktop (incl. download):

## sqlite
```                                                                                                                                                                                                                                                                                                         
taxit -vvv new_database  208.82s user 27.36s system 38% cpu 10:14.96 total
```

## postgres
```
 taxit -vvv new_database postgresql://dhoogest@127.0.0.1/ncbi_taxonomy_0_9_0  383.53s user 184.98s system 9% cpu 1:35:59.95 total
```

So, ~10 mins vs ~95 mins. The following info might be relevant for optimizing `executemany`:

-  https://docs.sqlalchemy.org/en/13/dialects/postgresql.html#psycopg2-fast-execution-helpers
-  https://hakibenita.com/fast-load-data-python-postgresql

**Comment by dhoogest** [comment]
@nhoffman if you get a shot, mind summarizing your comments related to this issue from a couple weeks ago when we discussed? If I recall it had something to do with the way in which postgres handles indices during insert (was not related to the `executemany` hypothesis proposed above).  I think the proposal for was to drop indicies on name nodes, then insert, then add the indices back.

**Comment by crosenth** [comment]
Postgres unique index inserts are slow.  I will look into a better strategy to add indexing after inserting rows.

**Comment by crosenth** [comment]
Adding @nhoffman

**Comment by crosenth** [comment]
@dhoogest - Strange that my local Postgres experiment took only 9 mins while yours took 90

**Comment by crosenth** [comment]
Also my local sqlite experiments took between one and three minutes on my laptop and Unicorn:

![Screenshot from 2020-08-18 09-23-20](https://user-images.githubusercontent.com/1760683/90539557-d5d0c780-e134-11ea-84e3-a19ad68acb43.png)

![Screenshot from 2020-08-18 09-24-58](https://user-images.githubusercontent.com/1760683/90539507-c3568e00-e134-11ea-9348-d27c9fdfd1cb.png)

**Comment by crosenth** [comment]
And lastly, here is my Unicorn Postgres experiment to db3:

![Screenshot from 2020-08-18 08-34-51](https://user-images.githubusercontent.com/1760683/90546215-9b6c2800-e13e-11ea-9b26-5fd8b098c72e.png)

**Comment by nhoffman** [comment]
I think we've addressed this to the best of our ability in v0.10.0 - feel free to reopen if you have specific ideas for further improvements.
